examples/ArduinoUnoNano-basic-Australia-AltSoftSerial/ArduinoUnoNano-basic-Australia-AltSoftSerial.ino

```diff
@@ -0,0 +1,157 @@
+/*
+ * Author: JP Meijers
+ * Edited: Lachlan Etherton
+ * Date: 2018-12-28
+ * Previous filename: TTN-Mapper-TTNEnschede-V1
+ *
+ * This program is meant to be used with an Arduino UNO or NANO, conencted to an RNxx3 radio module for the AU915 frequency plan.
+ * It will most likely also work on other compatible Arduino or Arduino compatible boards, like The Things Uno, but might need some slight modifications.
+ *
+ * Transmit a one byte packet via TTN. This happens as fast as possible, while still keeping to
+ * the 1% duty cycle rules enforced by the RN2483's built in LoRaWAN stack. Even though this is
+ * allowed by the radio regulations of the 868MHz band, the fair use policy of TTN may prohibit this.
+ *
+ * CHECK THE RULES BEFORE USING THIS PROGRAM!
+ *
+ * CHANGE ADDRESS!
+ * Change the device address, network (session) key, and app (session) key to the values
+ * that are registered via the TTN dashboard.
+ * The appropriate line is "myLora.initABP(XXX);" or "myLora.initOTAA(XXX);"
+ * When using ABP, it is advised to enable "relax frame count".
+ *
+ * Connect the RN2xx3 as follows:
+ * RN2xx3 -- Arduino
+ * Uart TX -- 9
+ * Uart RX -- 8
+ * Reset -- 12
+ * Vcc -- 3.3V
+ * Gnd -- Gnd
+ *
+ * If you use an Arduino with a free hardware serial port, you can replace
+ * the line "rn2xx3 myLora(mySerial);"
+ * with     "rn2xx3 myLora(SerialX);"
+ * where the parameter is the serial port the RN2xx3 is connected to.
+ * Remember that the serial port should be initialised before calling initTTN().
+ * For best performance the serial port should be set to 57600 baud, which is impossible with a software serial port.
+ * If you use 57600 baud, you can remove the line "myLora.autobaud();".
+ *
+ */
+#include <rn2xx3.h>
+#include <AltSoftSerial.h>
+
+AltSoftSerial mySerial;
+
+// create an instance of the rn2xx3 library,
+// giving the software serial as port to use
+rn2xx3 myLora(mySerial);
+
+// the setup routine runs once when you press reset:
+void setup()
+{
+  //output LED pin
+  pinMode(13, OUTPUT);
+  led_on();
+
+  // open serial communications and wait for port to open:
+  Serial.begin(57600); //serial port to computer
+  mySerial.begin(9600); //serial port to radio
+  Serial.println("It's start'n up mate!");
+
+  initialize_radio();
+
+  // transmit a startup message
+  myLora.tx("Oi, you're using a TTN Australian node!");
+
+  // setting the frequency plan to AU915
+  myLora.setFrequencyPlan(TTN_AU);
```

Comment:
Good point. Sounds like a plan to put it inside the function. Seeings that this sorts things out for the AU and US frequencies, is there any reason why we shouldn't put it in all the examples as an additional precaution?

Reply:
Yes maybe good to add it to some. I like keeping examples very minimal, and seeing as the Rn2483 works fine with its default settings in Europe, there is no point in calling this function. Also this function is unnecessary when joining via OTAA, as the network configures the other non-default channels. It's only in the US and AU cases where defining the FP is useful so that a join will be done on the correct channels.